mobile/native/src/ln_dlc/mod.rs

```diff
@@ -1008,6 +1008,16 @@ pub fn estimated_funding_tx_fee() -> Result<Amount> {
     Ok(fee)
 }
 
+/// Returns true if the provided address belongs to our wallet and false otherwise.
+/// Returns and error if the address is invalid.
+pub fn is_address_mine(address: &str) -> Result<bool> {
```

Comment:
Note: this might return a false-negative if the wallet has not generated enough addresses, i.e. if you recover your wallet and did not sync up to the questioned address this will return false.

Reply:
I will leave a comment.